src/libs/DistanceRequestUtils.ts

```diff
@@ -362,8 +362,8 @@ function getRate({
     }
     const policyCurrency = policy?.outputCurrency ?? getPersonalPolicy()?.outputCurrency ?? CONST.CURRENCY.USD;
     const defaultMileageRate = getDefaultMileageRate(policy);
-    const customUnitRateID = getRateID(transaction);
-    const customMileageRate = customUnitRateID ? mileageRates?.[customUnitRateID] : defaultMileageRate;
+    const customUnitRateID = getRateID(transaction) ?? String(CONST.DEFAULT_NUMBER_ID);
```

Comment:
We need to avoid using String(CONST.DEFAULT_NUMBER_ID) as per https://github.com/Expensify/App/issues/55814, maybe we can replicate with this

https://github.com/Expensify/App/blob/bfb97c55fabf1a820afe0a2b4b9699220331e953/src/pages/iou/request/step/IOURequestStepDistance.tsx#L154-L155